test/configuration/configuration.test.ts

```diff
@@ -37,6 +39,27 @@ suite('Configuration', () => {
     assert.deepStrictEqual(normalizedKeybinds[0].after, ['o', '<Esc>', 'k']);
   });
 
+  test.only('textwidth is configurable per-language', async () => {
+    const globalVimConfig = vscode.workspace.getConfiguration('vim');
+    const jsVimConfig = vscode.workspace.getConfiguration('vim', { languageId: 'javascript' });
+
+    try {
+      assert.equal(jsVimConfig.get('textwidth'), 80);
```

Comment:
Use `assert.strictEqual` or replace the import with

```typescript
import { strict as assert } from 'assert';
```

Reply:
Updated in [Use strictEqual](https://github.com/VSCodeVim/Vim/pull/7747/commits/cb07ddcc1c56f8c7a4f3dc06ed35c2c92d7ab83a)